bower.json

```diff
@@ -8,6 +8,7 @@
     "bedrock-angular": "^2.0.0",
     "bedrock-angular-alert": "^2.0.0",
     "bedrock-angular-modal": "^3.0.0",
-    "bedrock-angular-resource": "^2.0.0"
+    "bedrock-angular-resource": "^2.0.0",
+    "forge": "~0.6.34"
```

Comment:
forge seems unnecessary (not directly used) but `bedrock-angular-key` should be here.